docs/perl6.html

```diff
@@ -757,7 +757,7 @@ <h2 id="org9794805">ß - We need to talk about Unicode</h2>
 <p>
 This does not belong to the language proper, but it is still something
 that has to be mentioned. Languages are used to write applications,
-and these can use any language in the world. <a href="https://unicode-table.com/es/">Unicode</a> is the way to use
+and these can use any language in the world. <a href="https://symbl.cc/es/">Unicode</a> is the way to use
```

Comment:
Maybe while we're there change the default language to `en`? Or just leave it without any language.